modules/common/src/main/java/org/eclipse/xpanse/common/openapi/OpenApiUtil.java

```diff
@@ -129,6 +130,29 @@ public boolean downloadClientJar(String workdir) throws IOException {
         return true;
     }
 
+    /**
+     * Get the openapi-generator-cli.jar from target path.
+     *
+     * @return File  The openapi-generator-cli.jar path.
+     */
+    public File getClientJarFromTargetPath() {
+        String jarFileName = "openapi-generator-cli.jar";
+        String rootPath = System.getProperty("user.dir");
+        int modulesIndex = rootPath.indexOf("modules");
+        if (modulesIndex > 1) {
+            rootPath = rootPath.substring(0, modulesIndex);
+        }
+        String filePath;
+        if (rootPath.contains("runtime") && rootPath.contains("target")) {
```

Comment:
just confused about this implementation. 
If the openapi-generator-cli.jar is inside the final run-time jar.. then there is no need to check in target folder and the target folder does not even exist when the jar is compiled. 

What you must check is.. copy the runtime jar file to some other location.. and see if the runtime jar automatically uses the file from the "resources" inside jar.

Reply:
@swaroopar  Because we need to consider Windows/Linux and testing classes, the rootPaths they obtain here are different